scripts/deploy-all.ts

```
@@ -6,6 +6,11 @@ async function main() {
   const targetNetwork = parseArg("network");
   const ethers = await getEthers(targetNetwork);
   const deployer = await getDeployer(ethers);
+  const quorumBps = Number(process.env.QUORUM_BPS ?? "6700");
```

maybe better to throw an error here if nothing is configured in env instead of falling back to hardcoded default?

---

okay will fix this

---

https://github.com/ssvlabs/ssv-network/pull/343/commits/525a4f96a3c529bbda538f22a02836cd41adce35